.tina/schema.ts

```diff
@@ -39,3 +43,41 @@ export default defineSchema({
     },
   ],
 });
+
+// ===============
+
+// Tina config
+
+const branch = "main";
+// When working locally, hit our local filesystem.
+// On a Vercel deployment, hit the Tina Cloud API
+const apiURL =
+  process.env.NODE_ENV == "development"
+    ? "http://localhost:4001/graphql"
+    : `https://content.tinajs.io/content/${process.env.NEXT_PUBLIC_TINA_CLIENT_ID}/github/${branch}`;
+
+export const tinaConfig = defineConfig({
+  cmsCallback: (cms) => {
+    cms.flags.set("tina-admin", true);
```

Comment:
Do we still need this line?

Reply:
Correct 👍 We did need it, for the latest release, we don't.